src/IotHub/IotHub/IotHub/AddAzureRmIotHubKey.cs

```diff
@@ -27,9 +27,22 @@ namespace Microsoft.Azure.Commands.Management.IotHub
     [OutputType(typeof(PSSharedAccessSignatureAuthorizationRule))]
```

Comment:
@anusapan you'll need to set the `DefaultParameterSetName` in the above `Cmdlet` attribute since you now have multiple parameter sets for this cmdlet. It should be set to `ResourceParameterSet` since that was the previous default (only) parameter set.

Reply:
Done